src/runner/nunit.runner.Droid/project.json

```diff
@@ -1,11 +1,11 @@
 ﻿{
   "dependencies": {
-    "NUnit": "3.6.1",
```

Comment:
I don't know if this works, but https://stackoverflow.com/a/42336241/1768779 suggests that you can specify `"NUnit" : "[3.8.1]"` - to force compile against a specific version.

This is out of scope of this PR really, but as it needs changes anyway - would you mind testing it out? 😄  Ideally, I think we'd want to make it not run at all against any other version of NUnit, as doing so could lead to unexpected errors at runtime - and better to prevent them at compile time, the find them at runtime.